scripts/itest_parallel.sh

```diff
@@ -6,13 +6,19 @@ TRANCHES=$2
 TEST_FLAGS=$3
 ITEST_FLAGS=$4
 
+# Here we also shift 4 times and get the rest of our flags to pass on in $@.
+shift
```

Comment:
nit: could be `shift 4`.

Reply:
Changed to `shift 2` since I eliminated `TEST_FLAGS` and `ITEST_FLAGS`